app/assets/stylesheets/govuk_publishing_components/components/_metadata.scss

```diff
@@ -0,0 +1,70 @@
+.gem-c-metadata {
+  @include core-16;
+  @include responsive-bottom-margin;
+  @extend %contain-floats;
+
+  &.direction-rtl {
+    direction: rtl;
+    text-align: start;
+  }
+
+  .gem-c-metadata__term {
+    margin-top: 0.5em;
+    line-height: normal;
+
+    @include media(tablet) {
+      float: left;
+      clear: left;
+      width: 30%;
+      box-sizing: border-box;
+      max-width: 11em;
+      padding-right: $gutter-one-third;
+      margin-top: 0;
+    }
+  }
+
+  &.direction-rtl .gem-c-metadata__term {
+    float: right;
+    clear: right;
+
+    @include media(tablet) {
+      padding-left: $gutter-one-third;
+      padding-right: 0;
+    }
+  }
+
+  .gem-c-metadata__definition {
+    line-height: normal;
+
+    @include media(tablet) {
+      float: left;
+      width: 70%;
+    }
+
+    .gem-c-metadata__definition-link {
+      text-decoration: none;
+    }
+
+    .gem-c-metadata__definition-link:focus,
+    .gem-c-metadata__definition-link:active,
+    .gem-c-metadata__definition-link:hover {
+      text-decoration: underline;
+    }
+  }
+
+  &.direction-rtl .gem-c-metadata__definition {
+    float: right;
+  }
+
+  .gem-c-metadata__toggle-wrap {
+    display: none;
+
+    a {
```

Comment:
Is it possible to replace this with an explicit class?

Reply:
Removed the class.